README.md

```diff
@@ -158,6 +159,25 @@ Under [examples folder][examples-folder] we're documenting the testing cases whi
 
 ## FAQ
 
+### How do I mock API calls in my data fetching methods?
+
+Because `next-page-tester` isolates modules between "client" and "server" context mocks that are created in test (client context) wont execute in data fetching methods (server context).
```

Comment:
```suggestion
Since `next-page-tester` isolates modules between "client" and "server" context, mocks created in test (client context) won't execute in data fetching methods (server context).
```